liquibase-core/src/main/java/liquibase/integration/commandline/Main.java

```diff
@@ -1842,7 +1845,17 @@ protected void doMigration() throws Exception {
 
             try {
                 if (COMMANDS.UPDATE.equalsIgnoreCase(command)) {
-                    liquibase.update(new Contexts(contexts), new LabelExpression(getLabelFilter()));
+                    try {
+                        liquibase.update(new Contexts(contexts), new LabelExpression(getLabelFilter()));
+                    } catch (MigrationFailedException e) {
+                        if (!e.containsFailedChangeset() || !rollbackOnError) {
+                            throw e;
+                        }
+                        LicenseServiceUtils.checkProLicenseAndThrowException(command, ROLLBACK_ON_ERROR.getName());
```

Comment:
You probably don't need this license check here, since there is a license check inside the rollback one changeset command in the pro repo.

Reply:
True!